src/ResourceManager/ServiceFabric/Commands.ServiceFabric/help/New-AzureRmServiceFabricCluster.md

```diff
@@ -61,9 +61,9 @@ $clusterloc="SouthCentralUS"
 $subname="{0}.{1}.cloudapp.azure.com" -f $RGname, $clusterloc
```

Comment:
@juhacket a note about the markdown files in general: it looks like the markdown files weren't regenerated after some public API changes were made across a few cmdlets. Would you mind refreshing them?

Reply:
Regenerated the help.